tv.plex.PlexDesktop.yml

```diff
@@ -7,6 +7,7 @@ finish-args:
   - --socket=x11
   - --socket=pulseaudio
   - --device=all
+  - --device=dri
```

Comment:
We were asked to remove this during the flathub review of our flatpaks because `all` supposedly includes this. The docs are a bit unclear. @nedrichards (sorry for pinging out of the blue) can you advise?

@NightHammer1000 can you share application logs (in a gist maybe) from before and after enabling this? Find them in `~/.var/tv.plex.PlexDesktop/data/plex/Logs`.